src/test/java/hudson/plugins/openid/OpenIdLoginServiceTest.java

```diff
@@ -25,96 +25,102 @@
 
 import com.gargoylesoftware.htmlunit.ElementNotFoundException;
 import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlElement;
 import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
 import hudson.model.User;
 import hudson.security.GlobalMatrixAuthorizationStrategy;
 import hudson.security.HudsonPrivateSecurityRealm;
 import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
 import org.jvnet.hudson.test.Bug;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.JenkinsRule.WebClient;
 
 import java.io.IOException;
 import java.util.List;
 
 import static hudson.plugins.openid.OpenIdTestService.*;
+import static org.junit.Assert.*;
 
 /**
  * @author Paul Sandoz
  */
 public class OpenIdLoginServiceTest extends OpenIdTestCase {
-    HudsonPrivateSecurityRealm realm;
+    public static HudsonPrivateSecurityRealm realm;
 
-    @Override
-    public void setUp() throws Exception {
-        super.setUp();
-        realm = new HudsonPrivateSecurityRealm(false, false, null);
-        jenkins.getDescriptorByType(OpenIdLoginService.GlobalConfigurationImpl.class).setEnabled(true);
-    }
+    @Rule
+    public JenkinsRule jr = new LoginServiceTestRule();
 
     @Bug(9792)
```

Comment:
This annotation is obsolete. Makes sense to switch to <code>@Issue</code> during this bulk rework